.github/workflows/main.yml

```diff
@@ -16,6 +16,8 @@ jobs:
       - name: install
         run: |
           bun install
+        with:
+          bun-version: 1.1.13
```

Comment:
This is for github CI build step to work. This change is already on `dev`, there's some issues with bun latest versions so using 1.1.13 for now.